### PR TITLE
[Keyboard] Add Valor FRL TKL (+refactor)

### PR DESCRIPTION
**Pull request by Xelus22**
## Description

Add Valor FRL TKL.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by zvecr — CHANGES_REQUESTED** [review]
Due to the changes of `valor` being refactored to `valor/rev1` (both the change in build target, and moving of keymaps folder), this is current considered a "breaking change" and will need to target develop.

**Comment by Erovia** [comment]
Heya! Could you please clean up this PR?
It currently has 151 commits from all over the place and 804 modified files.

I think rebasing your changes to origin/master and force-pushing would be the easiest.
You can reach out to me on Discord if you need help.

**Review by zvecr — CHANGES_REQUESTED** [review]
Cheers for sorting that develop target stuff!

File: keyboards/xelus/valor/frl_tkl/readme.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Valor FRL TKL
+
```
Comment (Erovia) [inline keyboards/xelus/valor/frl_tkl/readme.md:2]: Do you have a pic about the PCB and the board by any chance?

File: keyboards/xelus/trinityxttkl/rules.mk [hunk 2]
```diff
@@ -15,6 +15,5 @@ NKRO_ENABLE = yes	    # USB Nkey Rollover
 BACKLIGHT_ENABLE = no
 RGBLIGHT_ENABLE = no
 
-
```
Comment (Erovia) [inline keyboards/xelus/trinityxttkl/rules.mk:18]: This seems to be a file for an unrelated board?

File: keyboards/xelus/valor/frl_tkl/readme.md (hunk 1 above)
Comment (Xelus22) [reply keyboards/xelus/valor/frl_tkl/readme.md:2]: image

I only have a KLE currenlty.

File: keyboards/xelus/trinityxttkl/rules.mk (hunk 2 above)
Comment (Xelus22) [reply keyboards/xelus/trinityxttkl/rules.mk:18]: Ah yes sorry. I will undo this. Was an accident

**Review by drashna — COMMENTED** [review]
I think that you don't need the full ChibiOS files.  Just what you've changed from default, and `#include_next "name"`, but @tzarc would know best.

**Comment by Xelus22** [comment]
I dont believe the chibiOS upgrade from tzarc is here yet. Though that is coming soon.

**Comment by Xelus22** [comment]
I feel like I should change this PR to make this a new keyboard as it is a different layout than the original valor keyboard. I also have plans for revisions of valor and hence should re-factor valor for future revisions such as the dawn60 revision in #10584 .

Any thoughts?

**Comment by drashna** [comment]
It may be a good idea, yeah, as it may make less work for you later on.

**Comment by tzarc** [comment]
> I dont believe the chibiOS upgrade from tzarc is here yet. Though that is coming soon.

Doesn't need to be here -- the ChibiOS upgrade isn't tightly coupled to config migrations. 

That said, no need to concern yourself with making the mods -- I'll just add this board to the conversion list while I prep all the changes across the repo.

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**